react-form-controls/my-app/src/RegistrationFormControlled.jsx

```diff
@@ -0,0 +1,43 @@
+import { useState } from 'react';
+export default function RegistrationFormControlled() {
+  const [isUsername, setUsername] = useState('');
```

Comment:
No need to update for now, but using "is" as in  `isUsername` is typically used to denote a boolean by many developers as a shorthand! 

I recommend sticking to `username` and `password` next time! 💡